### DISCUSSION TITLE
determine current version?

### DISCUSSION THREAD

**Discussion by kai-uwe-rommel** [TandoorRecipes/recipes#3388] [General]
How do I determine what version of Tandoor I am currently running?
In the UI, I do nowhere find any hint of some "about" information ...

**Comment by smilerz** [comment]
System

**Reply by kai-uwe-rommel** [reply]
What do you mean with this single word?

**Reply by smilerz** [reply]
![image](https://github.com/user-attachments/assets/8f16ceb8-c2c6-4408-8e3b-e10eae171c57)

**Reply by kai-uwe-rommel** [reply]
Ah, thanks. I was logged in as a regular user. You need to be logged in as admin/root to see this menu entry.

Well, it does not show the version there either.
I only see "commit d42d784aeb0595bbc0ed47eb8bbc27ba61b0787d" and "Date:   Tue Aug 29 13:09:38 2023 +0200".
The "Web Site - Commit" link goes to a GitHub page. I see indications there that I might be running 1.5.6 :-)

**Reply by smilerz** [reply]
you seem to be running the development branch - it doesn't have versions, just nightly commits

**Reply by kai-uwe-rommel** [reply]
I used the "docker compose" installation method following the docs. I use whatever container images that pulled at the time I did this ... Nov. 2023.

**Reply by kai-uwe-rommel** [reply]
I also gather from the docs and the release notes of subsequent releases that upgrading may not be easy due to a required Postgres upgrade (my current deployment uses Postgres 11 ... see previous comment).

**Reply by smilerz** [reply]
It's not really all that hard - just [upgrade postgres](https://docs.tandoor.dev/system/updating/#postgresql) first.

One note not included in the instructions - comment out the first two lines that create the user/group in the export.

**Reply by kai-uwe-rommel** [reply]
I just looked at an export of my database ... do you mean the create role/alter role lines in it?

**Reply by smilerz** [reply]
correct

**Reply by kai-uwe-rommel** [reply]
Why does that have to be removed?

**Reply by smilerz** [reply]
The compose takes care of it and it will occasionally lead to an error.
